src/main/java/net/minecraftforge/client/ClientCommandHandler.java

```diff
@@ -278,26 +302,34 @@ public static boolean sendMessage(String sendMessage)
         }
         catch (CommandSyntaxException syntax)// Usually thrown by the CommandDispatcher
         {
-            Minecraft.getInstance().player.sendMessage(
-                    new TextComponent("").append(ComponentUtils.fromMessage(syntax.getRawMessage())).withStyle(ChatFormatting.RED), Util.NIL_UUID);
-            if (syntax.getInput() != null && syntax.getCursor() >= 0)
+            if (syntax.getType() == CommandSyntaxException.BUILT_IN_EXCEPTIONS.dispatcherUnknownCommand())
             {
-                int position = Math.min(syntax.getInput().length(), syntax.getCursor());
-                MutableComponent details = new TextComponent("")
-                        .withStyle(ChatFormatting.GRAY)
-                        .withStyle((style) -> style
-                                .withClickEvent(new ClickEvent(ClickEvent.Action.SUGGEST_COMMAND, reader.getString())));
-                if (position > 10)
-                {
-                    details.append("...");
-                }
-                details.append(syntax.getInput().substring(Math.max(0, position - 10), position));
-                if (position < syntax.getInput().length())
+                // in case of unknown command, let the server try and handle it
+                Minecraft.getInstance().player.chat(sendMessage);
```

Comment:
Is there a specific reason why this calls `LocalPlayer#chat` directly instead of returning `false` from this method and letting vanilla call it instead?